tests/test_pattoo_shared/installation/test_packages.py

```diff
@@ -83,34 +83,13 @@ def test_install(self):
             result = expected_package in installed_packages
             self.assertEqual(result, expected)
 
-    def test_get_installed_packages(self):
-        """Unittest to test the get_installed packages function."""
-        packages = shared.run_script('python3 -m pip freeze')[1]
-
-        # Get packages with versions removed
-        pkg_names = [
-            package.decode().split('==')[0] for package in packages.split()
-            ]
-        pkg_versions = [
-            package.decode().split('==')[0] for package in packages.split()
-            ]
-        expected = dict(zip(pkg_names, pkg_versions))
-        result = get_installed_packages()
-        self.assertEqual(result, expected)
-
-    def test_version_check(self):
-        """Unittest to test the version_check function."""
-        # Test with outdated pattoo_shared version
-        with self.subTest():
-            shared.run_script('pip3 install PattooShared==0.0.89')
-            packages = get_installed_packages()
-            self.assertFalse(version_check('PattooShared', packages))
-
-        # Test by updating pattoo shared
-        with self.subTest():
-            shared.run_script('pip3 install PattooShared -U')
-            packages = get_installed_packages()
-            self.assertTrue(version_check('PattooShared', packages))
+    def test_get_package_version(self):
+        """Unittest to test the get_package_version function."""
+        package = 'PattooShared'
+        shared.run_script('python3 -m pip install {}==0.0.90'.format(package))
```

Comment:
After you run this unittest the pattoo developer's PattooShared version is 90. How will this be rectified after successfully running the script? Reinstall the previous version? Install 90 to a temp directory?

Reply:
A virtual environment is set up in a temporary directory with the environment_setup function for the tests for the packages so that the unittests for the packages won't affect the user's environment.

```
    @classmethod
    def setUpClass(cls):
        """Declare class attributes for Unittesting."""
        cls.venv_dir = tempfile.mkdtemp()
        environment.environment_setup(cls.venv_dir)
```